aiogram/dispatcher/filters/builtin.py

```diff
@@ -562,3 +562,43 @@ def validate(cls, full_config: typing.Dict[str, typing.Any]) -> typing.Optional[
             return chat_id in self.chat_id
 
         return False
+
+
+class AdminFilter(Filter):
+    """
+    Checks if user is admin in a chat.
+    If chat_id is not set, the filter will check in the current chat (correct only for messages).
+    chat_id is required for InlineQuery.
+    """
+
+    def __init__(self, admin_chat_id: typing.Optional[int] = None, admin_current_chat: typing.Optional[bool] = False):
+        self.chat_id = admin_chat_id
+        self.check_current_chat = admin_current_chat
+
+    @classmethod
+    def validate(cls, full_config: typing.Dict[str, typing.Any]) -> typing.Optional[typing.Dict[str, typing.Any]]:
+        result = {}
+
+        if "admin_chat_id" in full_config:  # use prefix 'admin' to not conflict with IdFilter
+            result["admin_chat_id"] = full_config.pop("admin_chat_id")
+        if "admin_current_chat" in full_config:  # set True if need to check current chat
+            result["admin_current_chat"] = full_config.pop("admin_current_chat")
+
+        return result
```

Comment:
This part need to be discussed.
What do you think about combining this filter arguments in one?

For example:
``` python3
is_chat_admin=True  # In current chat
is_chat_admin=-1234567890  # In specified chat
is_chat_admin=[-1234567890, -5432109876]  # In one of specified chats
```

Reply:
Yeah, I think that will be great. I will make it